pycbc/types/timeseries.py

```diff
@@ -401,6 +401,31 @@ def save_to_wav(self, file_name):
         """
         scaled = _numpy.int16(self.numpy()/max(abs(self)) * 32767)
         write_wav(file_name, self.sample_rate, scaled)
+        
+    def psd(self, segment_duration, avg_method='median'):
```

Comment:
@spxiwh I was just matching the existing default, but I don't have an issue with changing this to median-mean.